bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/localConnectInverter.xml

```diff
@@ -113,6 +113,24 @@
 				<description>The sum of PV currents from all strings</description>
 			</channel>
 
+			<channel id="inverter-temperature1" typeId="inverter-temperature">
+				<label>Inverter Temperature 1</label>
```

Comment:
See my comment for the constant - same applies here (even more than on the constant layer as this is presented to the user)